### DISCUSSION TITLE
SSL handshake failed when using in python

### DISCUSSION THREAD

**Discussion by nitanmarcel** [planetscale/discussion#407] [Bug Reports] [answered]
I can't figure out how to connect to the database, which each attempt failing with the error from the title SSL handshake failed

I'm using [databases](https://github.com/encode/databases) and [orm](https://github.com/encode/orm) to connect to the database.
```python
import databases
database = databases.Database('mysql://ofpslt425rs3vfpwyi7h:pscale_pw_AAASSA1314123@eu-central.connect.psdb.cloud/default', ssl='verify-full')
```

**Comment by fooforge** [comment, marked as answer]
You will likely need to specify the SSL certificate authority (CA) and point your database driver to it in order for your server to be able to establish a secure connection successfully. A good starting point would be to read through [this guide in our documentation](https://planetscale.com/docs/concepts/secure-connections#certificate-authorities). I have also [blogged about SSL-centric issues](https://planetscale.com/blog/supports-notes-from-the-field) not too long ago.

If you're still running issues afterwards, please share the parts of your code that handle the connection as well as the full error message.
